pipelines/controller/src/main/resources/templates/index.html

```diff
@@ -188,6 +210,22 @@
           <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
         </div>
 
+        <div class="alert alert-danger alert-dismissible fade show"
+             th:if="${lastRunDetails ne null AND lastRunDetails.status == T(org.openmrs.analytics.PipelineManager.Status).FAILURE}"
```

Comment:
This is what I meant by the above comment. If we use string constants or simple a failure status boolean, we don't need to expose package names on the JS side.

Reply:
Agreed and done